scene/main/scene_tree.cpp

```diff
@@ -1230,6 +1230,21 @@ Array SceneTree::_get_nodes_in_group(const StringName &p_group) {
 	return ret;
 }
 
+Node *SceneTree::_get_first_node_in_group(const StringName &p_group) {
+	Map<StringName, Group>::Element *E = group_map.find(p_group);
+	if (!E) {
+		return nullptr;
```

Comment:
Nitpick but it's to match the comments on master.
```suggestion
		return nullptr; // No group.
```